src/components/Film/index.js

```diff
@@ -23,16 +24,17 @@ function Film() {
               <h2>{film.release_date}</h2>
             </div>
             {/*We should sent films.movie_watched */}
-            <div>
-              <FilmWatched filmWatched={true} />
-              <EmojisRate/>
+            <div className="interactionContainer">
+              <FilmWatched watched={watched} setWatched={setWatched}/>
```

Comment:
Is this change state visible on the home page? I mean, when the user goes back to the home page, Can she see her movies marked as "watched" on the home page as well? 
Otherwise, need to implement useContext to share this state change with the other components.

Reply:
when a new user enters to a movie, will see that the default value of the rates is 0 or "add" in the case of the emojis, (this value come from the DB) and, when that user marks as watched that movie, and assign a rate, those rates functions will send the value to the DB :v, so that when that user goes to the Menu :v, the film-card should consult those values and put them there, showing the user his/her rates in emojis/stars :v, we could utilize useContext, yeah, to have that "value" saved in just one place and consulted by those different pages :v

Reply:
@Hai5edfm @AreYouIvan Is this having in mind that for each movie the value is different? I mean, in this case we only have one general state, will that value change while we are iterating into the array movies?